wandb/sdk/launch/runner/kubernetes.py

```diff
@@ -169,6 +169,15 @@ def populate_pod_spec(
             pod_spec["nodeSelectors"] = resource_args.get("node_selectors")
         if resource_args.get("tolerations"):
             pod_spec["tolerations"] = resource_args.get("tolerations")
+        if resource_args.get("security_context"):
+            pod_spec["securityContext"] = resource_args.get("security_context")
+        if resource_args.get("volumes") is not None:
+            vols = resource_args.get("volumes")
+            if not isinstance(vols, list):
+                raise LaunchError("volumes must be a list of volume specifications")
+            pod_spec["volumes"] = []
```

Comment:
Is there a reason for copying over the volumes and volume_mounts this way rather than just e.g.
`pod_spec["volumes"] = vols`

Reply:
Ah, jeez. No there isn't this is from the originalk solution. Cleaning up